src/client/dfuse/SConscript

```diff
@@ -174,16 +175,17 @@ def scons():
         Import('base_env')
         il_env = base_env.Clone()
         il_env['CC'] = 'gcc'
+        il_env['CXX'] = None
```

Comment:
This feeds into what I put above, I need to check the logs but it might mean scons ends up using None as a command and checking what options it supports.